src/state/sagas/auth/loginStartSaga.js

```diff
@@ -36,6 +36,8 @@ function* run() {
   yield call(navigator.navigate, 'Main', 'App');
 
   yield put(authActions.loginSuccess());
+
+  trackEvent('Login::Success');
```

Comment:
I think it's better to use specialized [`logLogin`](https://github.com/corymsmith/react-native-fabric/blob/master/Answers.js#L72) method for this